test/parallel/test-vm-codegen.js

```diff
@@ -10,15 +10,20 @@ const WASM_BYTES = Buffer.from(
 
 
 function expectsError(fn, type) {
-  try {
-    fn();
-    assert.fail('expected fn to error');
-  } catch (err) {
-    if (typeof type === 'string')
-      assert.strictEqual(err.name, type);
-    else
-      assert(err instanceof type);
-  }
+  assert.throws(
+    () => {
+      fn();
+      assert.fail('expected fn to error');
+    },
+    (err) => {
+      if (typeof type === 'string') {
+        assert.strictEqual(err.name, type);
+      } else {
+        assert(err instanceof type);
+      }
+      return true;
+    }
+  );
```

Comment:
I just had a look at the call sites and it would actually be best to just inline `assert.throws`. That would reduce the code overhead and the indirection is not necessary.

In case the constructor is used as second argument, nothing else has to be changed. If the second argument is a string, it should just be replaced with `{ name: string }`.